### PR TITLE
feat: include `_airbyte_emitted_at` in all matviews (where applicable)

### PR DESCRIPTION
**Pull request by emily-flambe**
## Summary 

This PR modifies the definitions for materialized views in four syncs so that they include `_airbyte_emitted_at`:

- action_builder
- action_network
- empower
- freshdesk
- twilio

The only matviews for which I did NOT add `_airbyte_emitted_at` within the syncs listed above were a few in freshdesk that are defined by unnesting fields from a base table (`conversation_*_email.sql`).

## Testing 

I tested action_builder, action_network, empower, and freshdesk using prod data by running the partner dbt models locally, creating the matviews in our testing project.

Twilio required adding `_airbyte_emitted_at` to one of the CTA base models so I tested the dbt for both the CTA and partner models by running the dbt locally using our dev project.

## TODOs

After merging, we will need to rebuild the matviews in prod and (where applicable) dev, so that the new definitions get applied. Note that action_network delivers tables, not matviews (to be addressed in a later ticket), so we don't need to do anything special for those.

TODO: run dbt locally with `--full-refresh` for:

- [ ] action_builder (prod)
- [ ] empower (prod)
- [ ] empower (dev)
- [ ] freshdesk (prod)
- [ ] freshdesk (dev)
- [ ] twilio (prod)
- [ ] twilio (dev)

### CODE REVIEW

**Comment by emily-flambe** [comment]
@jitoquinto we have a request to include a timestamp reflecting the time of the sync in all our airbyte syncs. We had a handful of models that were omitting this from the final materialized views, so I went and added `_airbyte_emitted_at` to all of them where possible (I saw three matviews in `freshdesk` that were unnesting a base table, and I think it's fine to leave the timestamp field out of those for now).

This isn't in a JIRA ticket, just something I wanted to knock out quickly - will trigger the affected syncs in dev to make sure it all works 👀

File: dbt-cta/freshdesk/models/2_partner_matviews/canned_response.sql [hunk 1]
```diff
@@ -1 +1 @@
-SELECT * FROM {{ source('cta', 'canned_response_base') }}
\ No newline at end of file
+SELECT * FROM {{ source('cta', 'canned_response_base') }}  _airbyte_emitted_at,
```
Comment (jitoquinto) [inline dbt-cta/freshdesk/models/2_partner_matviews/canned_response.sql:1]: this  doesnt seem right 🤔

File: dbt-cta/freshdesk/models/2_partner_matviews/canned_response_folder.sql [hunk 2]
```diff
@@ -1 +1 @@
-SELECT * FROM {{ source('cta', 'canned_response_folder_base') }}
\ No newline at end of file
+SELECT * FROM {{ source('cta', 'canned_response_folder_base') }}  _airbyte_emitted_at,
```
Comment (jitoquinto) [inline dbt-cta/freshdesk/models/2_partner_matviews/canned_response_folder.sql:1]: same here

File: dbt-cta/freshdesk/models/2_partner_matviews/discussion_comment.sql [hunk 3]
```diff
@@ -1 +1 @@
-SELECT * FROM {{ source('cta', 'discussion_comment_base') }}
\ No newline at end of file
+SELECT * FROM {{ source('cta', 'discussion_comment_base') }}  _airbyte_emitted_at,
```
Comment (jitoquinto) [inline dbt-cta/freshdesk/models/2_partner_matviews/discussion_comment.sql:1]: and here

File: dbt-cta/freshdesk/models/2_partner_matviews/discussion_topic.sql [hunk 4]
```diff
@@ -1 +1 @@
-SELECT * FROM {{ source('cta', 'discussion_topic_base') }}
\ No newline at end of file
+SELECT * FROM {{ source('cta', 'discussion_topic_base') }}  _airbyte_emitted_at,
```
Comment (jitoquinto) [inline dbt-cta/freshdesk/models/2_partner_matviews/discussion_topic.sql:1]: ++

File: dbt-cta/freshdesk/models/2_partner_matviews/product.sql [hunk 5]
```diff
@@ -1 +1 @@
-SELECT * FROM {{ source('cta', 'product_base') }}
\ No newline at end of file
+SELECT * FROM {{ source('cta', 'product_base') }}  _airbyte_emitted_at,
```
Comment (jitoquinto) [inline dbt-cta/freshdesk/models/2_partner_matviews/product.sql:1]: ++

File: dbt-cta/freshdesk/models/2_partner_matviews/ticket_field.sql [hunk 6]
```diff
@@ -1 +1 @@
-SELECT * FROM {{ source('cta', 'ticket_field_base') }}
\ No newline at end of file
+SELECT * FROM {{ source('cta', 'ticket_field_base') }}  _airbyte_emitted_at,
```
Comment (jitoquinto) [inline dbt-cta/freshdesk/models/2_partner_matviews/ticket_field.sql:1]: ++

File: dbt-cta/freshdesk/models/2_partner_matviews/solution_folder.sql [hunk 7]
```diff
@@ -1 +1 @@
-SELECT * FROM {{ source('cta', 'solution_folder_base') }}
\ No newline at end of file
+SELECT * FROM {{ source('cta', 'solution_folder_base') }}  _airbyte_emitted_at,
```
Comment (jitoquinto) [inline dbt-cta/freshdesk/models/2_partner_matviews/solution_folder.sql:1]: ++

File: dbt-cta/freshdesk/models/2_partner_matviews/solution_category.sql [hunk 8]
```diff
@@ -1 +1 @@
-SELECT * FROM {{ source('cta', 'solution_category_base') }}
\ No newline at end of file
+SELECT * FROM {{ source('cta', 'solution_category_base') }}  _airbyte_emitted_at,
```
Comment (jitoquinto) [inline dbt-cta/freshdesk/models/2_partner_matviews/solution_category.sql:1]: ++

File: dbt-cta/freshdesk/models/2_partner_matviews/solution_article.sql [hunk 9]
```diff
@@ -1 +1 @@
-SELECT * FROM {{ source('cta', 'solution_article_base') }}
\ No newline at end of file
+SELECT * FROM {{ source('cta', 'solution_article_base') }}  _airbyte_emitted_at,
```
Comment (jitoquinto) [inline dbt-cta/freshdesk/models/2_partner_matviews/solution_article.sql:1]: ++

File: dbt-cta/freshdesk/models/2_partner_matviews/sla_policy.sql [hunk 10]
```diff
@@ -1 +1 @@
-SELECT * FROM {{ source('cta', 'sla_policy_base') }}
\ No newline at end of file
+SELECT * FROM {{ source('cta', 'sla_policy_base') }}  _airbyte_emitted_at,
```
Comment (jitoquinto) [inline dbt-cta/freshdesk/models/2_partner_matviews/sla_policy.sql:1]: ++

File: dbt-cta/freshdesk/models/2_partner_matviews/settings.sql [hunk 11]
```diff
@@ -1 +1 @@
-SELECT * FROM {{ source('cta', 'settings_base') }}
\ No newline at end of file
+SELECT * FROM {{ source('cta', 'settings_base') }}  _airbyte_emitted_at,
```
Comment (jitoquinto) [inline dbt-cta/freshdesk/models/2_partner_matviews/settings.sql:1]: ++

File: dbt-cta/freshdesk/models/2_partner_matviews/scenario_automation.sql [hunk 12]
```diff
@@ -1 +1 @@
-SELECT * FROM {{ source('cta', 'scenario_automation_base') }}
\ No newline at end of file
+SELECT * FROM {{ source('cta', 'scenario_automation_base') }}  _airbyte_emitted_at,
```
Comment (jitoquinto) [inline dbt-cta/freshdesk/models/2_partner_matviews/scenario_automation.sql:1]: ++

**Review by jitoquinto — CHANGES_REQUESTED** [review]
There are some files that the column got appended to the sql files even though they already have `select *`.

**Comment by emily-flambe** [comment]
Oop! Well this is why we have code review lol - I'll address today!

**Comment by emily-flambe** [comment]
@jitoquinto I fixed the bugaboo you pointed out - before marking this ready for review, though, I've decided I'd like to take a sec to get the impacted syncs working in dev. I think some or all of them are not currently running dbt successfully in dev, and we should definitely test before applying changes to matview definitions 🤔 and I'll need to run the dbt locally first using --full-refresh to get the matview definitions to update (right?)

**Comment by jitoquinto** [comment]
> @jitoquinto I fixed the bugaboo you pointed out - before marking this ready for review, though, I've decided I'd like to take a sec to get the impacted syncs working in dev. I think some or all of them are not currently running dbt successfully in dev, and we should definitely test before applying changes to matview definitions 🤔 and I'll need to run the dbt locally first using --full-refresh to get the matview definitions to update (right?)

yup i think so! could be worth a try to just run it and see if it just turns off auto-refresh without full-refresh tho?

**Comment by emily-flambe** [comment]
@jitoquinto just tested all of the syncs affected by this change (using dbt commands run locally - see PR description), everything seems to check out on my end! Mind giving this another look when you have a sec? 🙏

**Review by jitoquinto — APPROVED** [review]

**Pull request merged**